docs/multiple_commands.md

```diff
@@ -0,0 +1,40 @@
+# Multiple commands
+
+`pg_eventstore` implements `#multiple` method to allow you to make several different commands atomic. Example:
+
+```ruby
+PgEventstore.client.multiple do
+  unless PgEventstore.client.read(stream3, options: { max_count: 1, direction: 'Backwards' }).last&.type == 'Removed'
+    PgEventstore.client.append_to_stream(stream1, event1)
+    PgEventstore.client.append_to_stream(stream2, event2)
+  end  
+end
+```
+
+All commands inside a block either all succeed or all fail. This allows you to easily implement complex business rules. This, however, comes with a price of performance. The more you put in a single block - the higher chance it will have conflicts with other commands that come in parallel, thus increasing overall time to complete. **Thus, do not put more than needed in there.** You may still want to use it though as it could simplify your implementation.
```

Comment:
```suggestion
All commands inside a `multiple` block either all succeed or all fail. This allows you to easily implement complex business rules. However, it comes with a price of performance. The more you put in a single block, the higher the chance it will have conflicts with other commands run in parallel, increasing overall time to complete. **Because of this performance implications, do not put more events than needed in a `multple` block.** You may still want to use it though as it could simplify your implementation.
```